src/main/java/in/twizmwaz/cardinal/module/modules/kit/KitCollection.java

```diff
@@ -0,0 +1,101 @@
+package in.twizmwaz.cardinal.module.modules.kit;
+
+import in.twizmwaz.cardinal.GameHandler;
+import in.twizmwaz.cardinal.event.CycleCompleteEvent;
+import in.twizmwaz.cardinal.module.modules.filter.FilterModule;
+import in.twizmwaz.cardinal.module.modules.filter.FilterState;
+import org.bukkit.Material;
+import org.bukkit.entity.EnderPearl;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.HandlerList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KitCollection implements KitRemovable {
```

Comment:
I know i define all the methods, but i have to set all the ones that the Collection interface has, and i don't really feel like it's needed at all
Methods like contains, add/remove, toArray, stuff like that will most likely never get used in the KitCollection class, and if you want to do it, you need to specify if you want to do it on the subkits or on the parent KitCollection list